### PR TITLE
oCIS beta7 release notes

### PR DESCRIPTION
**Pull request by pmaier1**

### CODE REVIEW

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 1]
```diff
@@ -23,6 +23,21 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 7
+
+Infinite Scale 2.0.0 beta7 includes stability, security and performance improvements. Web Office does now respect the user's language setting and system administrators can recover a lost admin password.
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:28]: Web Office does **not** respect the user's language. We can now give the language to the app provider, but we didn't implement it in web, yet. So backend made it possible, but in web ui nothing is different.

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 2]
```diff
@@ -23,6 +23,21 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
...
+* It is now possible to reset a lost administrator password using the command `ocis idm resetpassword`. https://github.com/owncloud/ocis/issues/4084[ocis#4084] https://github.com/owncloud/ocis/pull/4365[ocis#4365]
+* Encryption and authentication settings have been added to the mail server configuration. https://github.com/owncloud/ocis/pull/4443[ocis#4443]
+* The authentication middleware has been rewritten to increase security and stability. https://github.com/owncloud/ocis/pull/4374[ocis#4374]
+* There's a new endpoint `/app/open-with-web` which allows native clients to open applications like ONLYOFFICE or Collabora Online directly in the web browser. https://github.com/owncloud/ocis/pull/4376[ocis#4367] https://github.com/cs3org/reva/pull/3143[cs3org/reva#3143]
+* Infinite Scale 'extensions' have been renamed to 'services' to better fit their actual job. https://github.com/owncloud/ocis/pull/4407[ocis#4407]
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:37]: That was already done for beta1. There were some leftovers that are fixed in this beta.

Comment (pmaier1) [reply modules/ROOT/pages/ocis_release_notes.adoc:37]: Yes, I know. But we didn't really mention it so far.. should I just remove it?

File: modules/ROOT/pages/ocis_release_notes.adoc (hunk 1 above)
Comment (pmaier1) [reply modules/ROOT/pages/ocis_release_notes.adoc]: changed

File: modules/ROOT/pages/ocis_release_notes.adoc (hunk 2 above)
Comment (pmaier1) [reply modules/ROOT/pages/ocis_release_notes.adoc]: removed

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 3]
```diff
@@ -23,6 +23,21 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
...
+
```
[the remark this answers is not in the document]
Comment (pmaier1) [reply modules/ROOT/pages/ocis_release_notes.adoc:39]: added

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
@kulmann if you are ok with the changes too, pls remove your "changes requested" and you can merge afterwards 😄

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 4]
```diff
@@ -23,6 +23,22 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
...
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 7
+
+Infinite Scale 2.0.0 beta7 includes stability, security and performance improvements. ownCloud Web has been condensed a bit and the cut/copy/paste feature has been reworked. Web Office can now be configured to use a certain language and system administrators can recover a lost admin password.
+
+The most prominent changes in **Infinite Scale 2.0.0 beta7** and ownCloud Web 5.7.0 comprise:
+
+* Files and folders can now be cut/copy/pasted across Spaces using either action buttons after selecting them or by using well-known keyboard shortcuts. The former copy/paste dialog has been removed. https://github.com/owncloud/web/pull/7309[web#7309]
+* The general Web interface has a more condensed appearance. https://github.com/owncloud/web/pull/7363[web#7363]
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:33]:
```suggestion
* The Web interface has a generally more condensed appearance. https://github.com/owncloud/web/pull/7363[web#7363]
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 5]
```diff
@@ -23,6 +23,22 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 7
+
+Infinite Scale 2.0.0 beta7 includes stability, security and performance improvements. ownCloud Web has been condensed a bit and the cut/copy/paste feature has been reworked. Web Office can now be configured to use a certain language and system administrators can recover a lost admin password.
+
+The most prominent changes in **Infinite Scale 2.0.0 beta7** and ownCloud Web 5.7.0 comprise:
+
+* Files and folders can now be cut/copy/pasted across Spaces using either action buttons after selecting them or by using well-known keyboard shortcuts. The former copy/paste dialog has been removed. https://github.com/owncloud/web/pull/7309[web#7309]
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:32]:
```suggestion
* Files and folders can now be cut/copy/pasted using either action buttons after selecting them or by using well-known keyboard shortcuts. The former copy/paste dialog has been removed. This enables copy/paste across spaces. Cut/paste across spaces is not supported at the moment. https://github.com/owncloud/web/pull/7309[web#7309]
```

**Review by kulmann — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 6]
```diff
@@ -23,6 +23,22 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 7
+
+Infinite Scale 2.0.0 beta7 includes stability, security and performance improvements. ownCloud Web has been condensed a bit and the cut/copy/paste feature has been reworked. Web Office can now be configured to use a certain language and system administrators can recover a lost admin password.
+
+The most prominent changes in **Infinite Scale 2.0.0 beta7** and ownCloud Web 5.7.0 comprise:
```
[the remark this answers is not in the document]
Comment (pmaier1) [reply modules/ROOT/pages/ocis_release_notes.adoc:30]: hmm.. I didn't mention all the rc versions in the previous betas either.. inconsistent.. ok

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Pull request merged**